integration/testrunner/db.go

```diff
@@ -173,10 +174,21 @@ func diffLogEntry(tb testing.TB, logEntry *loggingpb.LogEntry, requireJustificat
 		tb.Errorf("queryResult field %v is blank", "jsonPayload.method_name")
 	}
 
-	if requireJustification {
-		if _, ok := jsonPayloadInfo.Metadata.AsMap()["justification"]; !ok {
-			tb.Errorf("queryResult field %v is blank", "jsonPayload.metadata.justification")
-		}
+	checkJustification(tb, jsonPayloadInfo)
+}
+
+func checkJustification(tb testing.TB, jsonPayloadInfo *audit.AuditLog) {
+	tb.Helper()
+	justification, ok := jsonPayloadInfo.Metadata.AsMap()["justification"]
+	if !ok {
+		tb.Fatalf("queryResult field %v doesn't exist", "jsonPayload.metadata.justification")
+	}
+	b, err := json.Marshal(justification)
+	if err != nil {
+		tb.Fatalf("failed to marshal justification: %v", err)
+	}
+	if string(b) == "null" {
```

Comment:
Should we also need to check against empty string?

Reply:
I checked bq, looks like the default value is null. But I can't find official docs for it. To make it more robust, I will add empty string check here.